code/src/components/Album.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { MenuBtns } from "./MenuBtns";
+import { Artists } from "./Artists";
+
+export const Album = (props) => {
```

Comment:
If you destructure your props like this:

```jsx
export const AlbumCard = props => {
  const { title, image } = props
}
```

or like this:

```jsx
export const AlbumCard = ({ title, image }) => {
  //
}
```

you don't have to write the props parameter when using the props:
`<h1>{props.title}</h1>`
This is enough:
`<img src={image} alt="Album cover" />`